plugins/cocktail-lookup.js

```diff
@@ -0,0 +1,106 @@
+const Doorman = require('../../doorman');
```

Comment:
in fact, the modules are the exact same as with that bot, so we could just import those via the package.json and config as I did with Kirbi

that cool?